debian/control

```diff
@@ -7,5 +7,5 @@ Standards-Version: 3.9.6.1
 
 Package: batwatch
 Architecture: all
-Depends: adduser (>= 3.11), confighelper (>=0.3), gpgmailer (>=0.7), python-daemon, python-pydbus, upower
+Depends: adduser (>= 3.11), gpgmailer (>=0.7), parkbench-common (>=0.8), python-daemon, python-pydbus, upower
```

Comment:
Shouldn't gpgmailer also be greater than 0.8?

Reply:
Oops, yeah.